practice-01/index.html

```diff
@@ -292,14 +306,101 @@ <h4 class="card__footer">27 Dec 2022,<span class="card__footer--gray"> by Roxie
     <!-------------------
     footer 
     --------------------->
-    <footer>
-        <div class="container">
-            <div class="footer__content">
-                <div class="content__left"></div>
-                <div class="content__right"></div>
+    <footer class="footer">
+        <div class="footer__content container">
+            <div class="content__left">
+                <a href="#" class="logo">
+                    <img src="./img/logo.png" alt="logo">
+                </a>
+                <div class="footer__text">
```

Comment:
do we need footer__text

Reply:
No need, because we can give the tag font--small nested